_episodes/02-getting-started.md

```diff
@@ -41,6 +129,17 @@ $ cd hello-world
 ~~~
 {: .language-bash }
 
+### Using Git
+
+One of the main barriers to getting started with Git is the language. Although some of the language used in Git is
```

Comment:
I think this could be the section of the lesson that @doingarchives might have been referring to as having a negative tone. I agree, and I would like to change it, but I'm going to put that as a new issue. Focused here on getting the SSH authentication integrated into the lesson in time for the August 13 deadline.

Reply:
Understood - as a maintainer I like to see changes come from the user base (instructors / learners). The data we have on most granular subjects is sparse - I think substantive changes should be polled to either the advisory committee or a sample of users (or potential users). We'll not get statistical accuracy but we can triangulate some of the conversations. When the proposals are 'lateral' meaning maintainer to maintainer, I'll put 'eyes' (the emoji) on those pseudo issues and perhaps they can be adjudicated at a higher level (advisory) or with a poll of former learners / instructors and prospective learners/instructors.

Reply:
The narrow technical SSH issue can be fixed well before the deadline. It was embedded in a more thoughtful (and necessary) conversation.